helm/values.yaml

```diff
@@ -0,0 +1,265 @@
+  # Add or remove prefixes as you require.
+  gunPrefixes:
+  - "docker.io/"
+  - "mydomain.com/"
+
+
+  # Set 'trust: local' if you don't want to spin up a signer instance. Otherwise, the remote signer service is made
+  # available at the 'hostname' and 'port' specified here.
+  trust:
+    type: remote
+    hostname: notary-signer
+    port: 7899
+
+  # The credentials configuration for server storage
+  storageCredentials:
+    user: server
+    password: null
+
+    # If 'password' is null, then a secret name and key to discover the database credentials must be provided
+    passwordSecretName: server-db-password
+    passwordSecretKey: password
+
+signer:
+
+  # The number of signer replicas to create
+  replicas: 3
+
+  # The version of the image to use for the signer component.
+  version: signer-0.6.1-2
+
+  # The port to expose the signer service on. This is just for the internal service, it's not exposed externally
+  # even if 'ingress.enabled: true' or loadBalancer.enabled: true'.
+  port: 7899
+
+  alias:
+
+    # This is the value passed to the 'storage.default_alias' field of the signer configuration file.
+    defaultAlias: alias
+
+    # A default alias passphrase to set the 'NOTARY_SIGNER_<DEFAULT_ALIAS>' environment variable to.
+    # If this value is set, the chart will create a Kubernetes secret called 'signer-alias' and set
+    # this secret there. If this is not set, the chart will require secret configured in aliasSecretName.
+    aliasPassphrase: null
+
+    # The name and key of the secret used to set the 'NOTARY_SIGNER_<DEFAULT_ALIAS>' environment variable to.
+    # This should be a pre-existing Kubernetes secret provided by you, this chart won't create it.
+    aliasSecretName: signer-alias
+    aliasSecretKey: passphrase
+
+  # The credentials configuration for server storage
+  storageCredentials:
+    user: signer
+    password: null
+
+    # If 'password' is null, then a secret name and key to discover the database credentials must be provided
+    passwordSecretName: signer-db-password
+    passwordSecretKey: password
+
+logging:
+  # The logging level to set the 'logging.level' field in both the notary and signer config files.
+  level: debug
+
+tls:
+  # If this is set to true, the template will use the certificates and keys provided, otherwise it will automatically
+  # generate self-signed certificates based on the parameters passed to the 'generated' map.
+  # If using custom certificates, they should be base64-encoded. And both the public certificate and the private key
+  # must be provided.
+  custom: false
+
+  # The CA that signed the server/signer certificates. It's assumed that the same CA is used for bother server
+  # and signer certificates.
+  # It doesn't need to strictly be a root CA, it can be an intermediate one.
+  rootCACert: null
+
+  # The public certificate and corresponding private key for the server.
+  server:
+    cert: null
```

Comment:
If I want to use my own certificates, in what format should I put the certificate here? base64? string? I would be nice if specified in the comments. The same for the rest of the certificates.

Reply:
It's a string but yes, it should be clarified in the comments. Thanks!